lib/generators/stimulus_reflex/templates/app/javascript/packs/application.js.tt

```diff
@@ -0,0 +1,13 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+import Rails from "@rails/ujs"
+import Turbolinks from "turbolinks"
+import * as ActiveStorage from "@rails/activestorage"
```

Comment:
We should either drop `activestorage` here are include it in all installer-environments

Reply:
Agreed. I'm actually leaning towards dropping it. As I progress, I'm finding myself preferring to copy in "ideal" files while backing up old versions. I'm very proud of how this part turned out, with the list of changed files at the end.

After all, it's the SR installer, not the "AI that understands everything that could be in your code".

So if they are using AS, they can make sure it's there as they merge the suggested file and the `.bak` version.

Reply:
Same here, we should anyway not override the users `application.js` anyway. We just need to append 
```js
import "config"
import "controllers"
import "channels"
```
and then we also shouldn't need to care about `activestorage` or whatever else might be in the `application.js`